SITnett/templates/medlemmer/medlemmer.html

```diff
@@ -15,11 +16,11 @@ <h2>Medlemmer</h2>
 	</div>
 	<div class="card-wrapper">
 		{% for medlem in mliste %}
-		<a class="card" href="{% url 'medlem_info' medlem.id %}">
+		<a class="card card-{{medlem.get_undergjeng_display}}" href="{% url 'medlem_info' medlem.id %}">
```

Comment:
Dette gjør strengen til lowercase:
```suggestion
		<a class="card card-{{ medlem.get_undergjeng_display|lower }}" href="{% url 'medlem_info' medlem.id %}">
```